src/cpu/ref_softmax.cpp

```diff
@@ -170,7 +177,7 @@ status_t ref_softmax_fwd_t::execute_forward_dense(const exec_ctx_t &ctx) const {
             } else if (pd()->is_logsoftmax()) {
                 val = d - space_denom;
             }
-            val *= src_scales[0];
+            if (with_src_scales) val *= src_scales[0];
```

Comment:
Is the scale datatype checked anywhere? I would recommend to use load_float_value here.